competitions/100Change2020/ansible/inv/local/group_vars/all.tmpl

```diff
@@ -38,7 +38,7 @@ mediawiki_install_directory: ${MEDIAWIKI_INSTALL_DIRECTORY}
 
 # The password for the mediawiki user admin to run automated
 # commands against wikis (such as csv2wiki, setup, etc)
-mediawiki_admin_password: ${MEDIAWIKI_ADMIN__PASSWORD}
+mediawiki_admin_password: ${MEDIAWIKI_ADMIN_PASSWORD}
```

Comment:
But I need to know which underscore you removed!

Reply:
It's a secret